model/session.go

```diff
@@ -29,6 +29,7 @@ type Session struct {
 	LastActivityAt int64     `json:"last_activity_at"`
 	UserId         string    `json:"user_id"`
 	TeamId         string    `json:"team_id"`
+	TeamURL        string    `json:"team_url"`
```

Comment:
Why do I ned this if I have TeamId?  wonder if this should just be Name (see comment below).